### ISSUE TITLE
bad temperature set at print start

### ISSUE THREAD

**Issue by nicolasloyce** [malnvenshorn/OctoPrint-FilamentManager#163]
**Describe the bug**
when I'm using filament manager while printing a file from octoprint, temperature is randomly set to 525°c, and it gives to a max temp error. I've found the way to reproduce the bug all the time : 

**Steps to reproduce**
1. upload a file
2. if the job needs for exemple 150g set a spool that is 145g
3. a notification of insuficient spool is displayed
4. print anyway, -> temp is set to 525°c

**Expected behavior**
I'm expected the temperature set in the Gcode : 265°c for me.

**Did the same happen when all other 3rd party plugins are disabled?**
I didnt tryed to disable all my others plugins but when I disable this one + reboot it works again. 

Here is the log file : 
[octoprint.log.txt](https://github.com/malnvenshorn/OctoPrint-FilamentManager/files/3266239/octoprint.log.txt)

**Comment by malnvenshorn** [comment]
Wrong temperature offset?

And please don't copy and paste the log file it's simply unreadable. Just drag and drop the file.

**Comment by nicolasloyce** [comment]
OK, sorry, I didnt want to make it bad, I 've edit my message. 
Yes, you may call it like that ... but  that's not an offset, 
it's the wrong start temp set. 
It is set at 265°c and it started at 525°c

**Comment by malnvenshorn** [comment]
I'm referring to the temperature offset you can set in the spool menu (see screenshots). 

There was a similar report of a user who thought that the value is the absolute temperature used for printing instead of an offset. The result was a temperature error (ticket #149).

**Comment by amemo06** [comment]
Same for me. Still the step by step is even more simple and no need of an empty filament spool.

- Just select a job to print
- Select a different the filament spool from Filament manager on the main octoprint screen
- Start the print 

Bed will warm up fine
BUT the head heating request will be 400°. And it will really try to reach this temperature thus leading to no way to stop heating. Of course, filament will be burnt. Other issues could occurs if the printer is left unattended which is the main purpose of octoprint

Changing the print job do not help if you succeed due to the above constraint (temperature will not be reached).
Stop the job and restart don't help. Same comment as above
The only solution for me is to shutdown the printer and restart

**Comment by malnvenshorn** [comment]
Same question: Do you have a temperature offset set for your selected filament?

**Comment by amemo06** [comment]
Since you asked that question, I better understood your previous question and explanations.
Yes I have set the desired temperature per filament instead of a temperature offset.
That, then explains very well the issue I faced. 
Cura sends the requested temperature and Filament Manager adds the offset which was the same temperature thus leading to 2x200° 😭
All seems clear to me now.
Thanks for the explanations. 
I understand quickly if you explain me a long time 😁
